platform/run_image.go

```diff
@@ -78,31 +78,23 @@ func BestRunImageMirrorFor(targetRegistry string, runImageMD files.RunImageForEx
 
 	// Select the first run image we have access to
 	for _, image := range runImageMirrors {
-		ok, err := checkReadAccess(image, keychain)
-		if err != nil {
-			return "", err
-		}
-		if ok {
+		if ok, _ := checkReadAccess(image, keychain); ok {
```

Comment:
Ideally we'd log the error at this point, but that requires a logger, which is a bigger change